src/test/resources/unrc/dose/Bono.feature

```diff
@@ -0,0 +1,91 @@
+
+Feature: test based challenge equals
```

Comment:
The feature description is somewhat confusing, is this feature about an `equals` methods for test based challenges? If so, the scenario is not matching the feature.